Examples/NoModules/Chapter 21/Ex21_03/Ex21_03.cpp

```diff
@@ -0,0 +1,15 @@
+// Violating constraints of uninstantiated class members
+#include "Array.h"
+#include <memory>  // For std::unique_ptr<>
+
+// Assert that Array<std::unique_ptr<int>> is a valid type
+static_assert(requires { typename Array<std::unique_ptr<int>>; });
```

Comment:
doesn't compile on VC++. Bug in VC++ or bug in code?
```
1>Ex21_03.cpp(6,15): error C2059: syntax error: 'requires'
1>Ex21_03.cpp(6,24): error C2143: syntax error: missing ';' before '{'
1>Ex21_03.cpp(6,24): error C2143: syntax error: missing ')' before ';'
1>Ex21_03.cpp(6,24): error C2447: '{': missing function header (old-style formal list?)
1>Ex21_03.cpp(6,65): error C2059: syntax error: ')'
```

Reply:
Compiles fine with GCC, so I'm thinking not implemented yet in VC++?